File: Q27/Q27.py
def print_alternating_pyramid(n):
    for i in range(1, n + 1):
        print(" " * (n - i), end="")
        symbol = '*' if i % 2 == 1 else '#'
        print((symbol + " ") * i)

n = int(input("Enter the number of rows: "))
print_alternating_pyramid(n)

File: Q27/Q27.js
function printAlternatingPyramid(n) {
    for (let i = 1; i <= n; i++) {
        let line = " ".repeat(n - i);
        let symbol = i % 2 === 1 ? "*" : "#";
        line += (symbol + " ").repeat(i);
        console.log(line);
    }
}

const n = parseInt(prompt("Enter the number of rows: "));
printAlternatingPyramid(n);
